Cheers.java

```diff
@@ -1,6 +1,36 @@
 // Prints a crowd cheering output.
 public class Cheers {
         public static void main(String[] args) {
-	    // Replace this comment with your code
+            String nameInput = args[0]; 
+            String lowerCase = "abcdefghigklmnopqrstuvwxyz";
+            String upperCase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+            String capitalName = ""; 
+            int j = 0; 
+            while (j < nameInput.length()) {
+                char letter = nameInput.charAt(j);
+                if (lowerCase.indexOf(letter) == -1) {
```

Comment:
**THE SOLUTION LOGIC WITH THE GIVEN TOOLS IS EXCELLENT!!!!!!!!!!**
**REALLY GOOD WORK**